packages/build-info/src/frameworks/astro.ts

```diff
@@ -27,13 +27,18 @@ export class Astro extends BaseFramework implements Framework {
     dark: '/logos/astro/dark.svg',
   }
 
+  env = {
+    NODE_VERSION: '18',
```

Comment:
Mhm I think we should check if a node version is already set in the environment variables. What if a user wants to use node 20?

So we should first check if a node version is set that is larger than 18 and if an environment variable is set.

There are basically 2 ways to set a node version through the UI (should be available through the `this.project.getCurrentNodeVersion` and `this.project.getEnv('NODE_VERSION')` if the user set's the env variable we should not set it regardless of if it is greater or not

Reply:
Yeah thats a great point, just looked at how the other frameworks did it and went with that. I vaguely remember being told the user's will always be respected, but hey